test/unit/ngsiv2/expressions/expressionBasedTransformations-test.js

```diff
@@ -83,7 +83,7 @@ var iotAgentLib = require('../../../../lib/fiware-iotagent-lib'),
                         name: 'consumption_x',
```

Comment:
Only stetical/style fixes in this file in this PR?

Reply:
Yes, it's just to pass lint.

Reply:
NTC